crates/bevy_core_pipeline/src/auto_exposure/settings.rs

```diff
@@ -0,0 +1,100 @@
+use std::ops::RangeInclusive;
+
+use super::compensation_curve::AutoExposureCompensationCurve;
+use bevy_asset::Handle;
+use bevy_ecs::{prelude::Component, reflect::ReflectComponent};
+use bevy_reflect::Reflect;
+use bevy_render::{extract_component::ExtractComponent, texture::Image};
+use bevy_utils::default;
+
+/// Component that enables auto exposure for an HDR-enabled 2d or 3d camera.
+///
+/// Auto exposure adjusts the exposure of the camera automatically to
+/// simulate the human eye's ability to adapt to different lighting conditions.
+///
+/// Bevy's implementation builds a 64 bin histogram of the scene's luminance,
+/// and then adjusts the exposure based on that histogram. Using a histogram
```

Comment:
More details on "adjusts exposure based on that histogram" would be great. What's the ultimate goal here? Are we trying to keep the perceived brightness of the scene constant?